Setup/Methods/Delete.php

```diff
@@ -31,12 +31,16 @@ public function __construct(string $table)
      */
     public function __toString(): string
     {
-        return 'DELETE FROM ' . $this->table . ($this->conditions === []
+        return sprintf(
+            "DELETE FROM %s%s",
+            $this->table,
+            $this->conditions===[]
```

Comment:
Yoda style en missende spaties (request change), maar voor de leesbaarheid is het misschien ook mogelijk om dit buiten de return statement voor te bereiden en niet in een ternary te stoppen (ter overweging)? Edit: ik zie dat dit zo verderop ook weer is, wat mij betreft (zie ook mijn vorige goedkeuring) laat je dit allemaal zoals je het gevonden hebt. Maar als we het echt netjes doen, moeten overal spaties en yoda style gehanteerd worden (krijg je er geen waarschuwingen voor in phpStorm?)